Public/Src/Cache/ContentStore/Library/Stores/FileSystemContentStoreInternal.cs

```diff
@@ -669,15 +669,18 @@ protected override async Task<BoolResult> StartupCoreAsync(OperationContext cont
                 GetPinnedSize,
                 _nagleQueue);
 
+            var quotaKeeperConfiguration = QuotaKeeperConfiguration.Create(
+                Configuration,
+                _distributedEvictionSettings,
+                size,
+                _settings.StartPurgingAtStartup,

```

Comment:
>_settings [](start = 16, length = 9)

Why not just pass `_settings` here?

Reply:
God idea.

---
In reply to: [273230438](https://github.com/Microsoft/BuildXL/pull/102#discussion_r273230438) [](ancestors = 273230438)